static/js/marker.js

```diff
@@ -48,14 +61,14 @@ var MarkerView = Backbone.View.extend({
 			this.$el.find(".profile-image").attr("src", "/static/img/lamas.png");
 			this.$el.find(".profile-image").attr("width", "50px");
 //		}
-//  	this.$el.find(".type").text(TYPE_STRING[this.model.get("type")]);
+		this.$el.find(".type").text(TYPE_STRING[this.model.get("type")]);
 //		var display_user = "";
 //		if (user && user.first_name && user.last_name) {
 //			display_user = user.first_name + " " + user.last_name;
 //		} else {
 			display_user = 'הלשכה המרכזית לסטטיסטיקה';
 //		}
-		this.$el.find(".added-by").text("מקור: " + display_user);
+		this.$el.find(".added-by").text("נוסף על ידי " + display_user);
```

Comment:
I've used
git fetch origin
git fetch origin master
then committed
then pushed

shouldn't that make sure my clone is up to date and pushing my own commits?